### PR TITLE
revolver changes- non akimbo firerate buff, and knockback readded

### PR DESCRIPTION
**Pull request by Pariah919**
revolver changes

## About The Pull Request
Readds knockback to revolver, makes akimbo have the same firerate as before so no change. And readds knockback to the bullet.

## Why It's Good For The Game
Everyone has cooled off about revolver. Let it have SOME gimmick.
This should give it a neat gimmick without making it top tier overpowered.

## Changelog
:cl:
balance: Revolver fires at 0.15 when non akimboed, and has knockback again.
/:cl:

### CODE REVIEW

**Comment by turnop** [comment]
good changes, if le revolver man becomes too funny with these just make revolver not fit in backpacks or satchels

**Comment by Lumipharon** [comment]
Akimbo additional delay is a multiplier, not additive AFAIK.

File: code/modules/projectiles/guns/revolvers.dm [hunk 1]
```diff
@@ -174,7 +174,8 @@
 		/obj/item/attachable/shoulder_mount,
 	)
 	attachable_offset = list("muzzle_x" = 33, "muzzle_y" = 19,"rail_x" = 13, "rail_y" = 23, "under_x" = 22, "under_y" = 14, "stock_x" = 22, "stock_y" = 19)
-	fire_delay = 0.2 SECONDS
+	fire_delay = 0.15 SECONDS
+	akimbo_additional_delay = 0.05 SECONDS
```
Comment (Pariah919) [inline code/modules/projectiles/guns/revolvers.dm:178]:
```suggestion
	akimbo_additional_delay = 1.6 // Ends up as 0.249, so it'll get moved up to 0.25.
```

File: code/modules/projectiles/guns/revolvers.dm [hunk 2]
```diff
@@ -174,7 +174,8 @@
 		/obj/item/attachable/shoulder_mount,
 	)
 	attachable_offset = list("muzzle_x" = 33, "muzzle_y" = 19,"rail_x" = 13, "rail_y" = 23, "under_x" = 22, "under_y" = 14, "stock_x" = 22, "stock_y" = 19)
-	fire_delay = 0.2 SECONDS
+	fire_delay = 0.15 SECONDS
+	akimbo_additional_delay = 1.6 // Ends up as 0.249, so it'll get moved up to 0.25.
```
Comment (BraveMole) [inline code/modules/projectiles/guns/revolvers.dm:178]: 0.6

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/revolvers.dm (hunk 2 above)
Comment (Pariah919) [inline code/modules/projectiles/guns/revolvers.dm:178]:
```suggestion
	akimbo_additional_delay = 0.6 // Ends up as 0.249, so it'll get moved up to 0.25.
```

Comment (Pariah919) [reply code/modules/projectiles/guns/revolvers.dm]: done

**Comment by GravitronBlue** [comment]
> good changes, if le revolver man becomes too funny with these just make revolver not fit in backpacks or satchels

I'm way overdue thinking this, but would disabling quickdrawing weapons from storage containers curb the unga meta of just having a stockpile of revolver bags to magdump(cylinderdump?) with?

> Everyone has cooled off about revolver.

Only so long as the xeno hive's T1s aren't being shredded by Ocelot ungas on bikes.
Readding knockback could also mean a FUN return of flank cheese jetpack ungas.

Though frankly I don't mind the knockback so long as heavy/large xeno castes aren't being bullied around by it.

**Comment by Lumipharon** [comment]
@Pariah919 Keep in mind the default akimbo additional delay is 0.5, so increasing it to 0.6 is actually a very minor akimbo nerf (I need to change my PR as well...)
For comparison, akimbo ROF per gun is changing from 0.375 (rounds to 0.4 or 0.35? I unno) to 0.24, rounding to 0.25.

**Review by BraveMole — APPROVED** [review]

**Pull request merged**